docs(admission-controller): add chart migration howto (#808)
kubewarden/docs: docs/admission-controller/version-1.37/modules/en/pages/community-specific/howtos/chart-migration.adoc

= Migration from three-chart setup
:page-languages: [en, de, es, fr, ja, pt, zh]
:revdate: 2026-06-26
:page-revdate: {revdate}
:description: Migrate from the legacy kubewarden-crds, kubewarden-controller, and kubewarden-defaults Helm charts to the unified admission-controller chart.
:doc-persona: ["kubewarden-operator"]
:doc-topic: ["chart-migration"]
:doc-type: ["howto"]
:keywords: ["{admc-name}", "kubewarden", "helm", "chart", "migration", "upgrade", "kubewarden-crds", "kubewarden-controller", "kubewarden-defaults", "admission-controller"]
:sidebar_label: Migration from three-chart setup
:sidebar_position: 10
:current-version: {page-origin-branch}

Starting from Kubewarden 1.37, a single helm chart is used to manage
the stack.

If you are running the legacy three-chart setup (`kubewarden-crds`,
`kubewarden-controller`, `kubewarden-defaults`), you have upgrade paths:

* Use the migration script (`kubewarden-unified-adm-controller-chart-migration.sh`):
  this performs the migration without any downtime.
* Follow the reinstallation instructions: cleaner approach, but involves a short
  time during which your cluster won't be protected by policies.

[#manual-reinstallation]

> == Manual reinstallation

Reviewer: I would also add a section (h3) with the requirements, like kubectl, yq and helm (I think both v3 and v4 are fine in this case).

= Migration from three-chart setup
:page-languages: [en, de, es, fr, ja, pt, zh]
:revdate: 2026-06-26
:page-revdate: {revdate}
:description: Migrate from the legacy kubewarden-crds, kubewarden-controller, and kubewarden-defaults Helm charts to the unified admission-controller chart.
:doc-persona: ["kubewarden-operator"]
:doc-topic: ["chart-migration"]
:doc-type: ["howto"]
:keywords: ["{admc-name}", "kubewarden", "helm", "chart", "migration", "upgrade", "kubewarden-crds", "kubewarden-controller", "kubewarden-defaults", "admission-controller"]
:sidebar_label: Migration from three-chart setup
:sidebar_position: 10
:current-version: {page-origin-branch}

Starting from Kubewarden 1.37, a single helm chart is used to manage
the stack.

If you are running the legacy three-chart setup (`kubewarden-crds`,
`kubewarden-controller`, `kubewarden-defaults`), you have two possible upgrade paths:

* Use the migration script (`kubewarden-unified-adm-controller-chart-migration.sh`):
  this performs the migration without any downtime.
* Follow the reinstallation instructions: cleaner approach, but involves a short
  time during which your cluster won't be protected by policies.

[WARNING]

> ====
> It's important to remember that, as described at
> xref:/reference/upgrade-path.adoc[upgrade path documentation],  version jumping
> is not allowed. You must migration from {admc-short-name} v1.36 to v1.37
> ====

Reviewer 2: ```suggestion
====
Remember that, as described at
xref:/reference/upgrade-path.adoc[upgrade path documentation],  version jumping
isn't allowed. You must migrate from {admc-short-name} v1.36 to v1.37
====
```
